src/main/java/no/fint/sikri/data/noark/korrespondansepart/KorrespondansepartFactory.java

```diff
@@ -18,6 +19,10 @@
 
 @Service
 public class KorrespondansepartFactory {
+
+    @Value("${fint.sikri.droppinternekorrespondanseparter:false}")
```

Comment:
Convention for property naming is English, and chaining words with hyphens.

I'd suggest `fint.sikri.skip-internal-contacts`.

Reply:
Updated